Library/OcCpuLib/OcCpuLib.c

```diff
@@ -884,27 +880,33 @@ ScanAmdProcessor (
         //
         if (Cpu->MaxExtId >= 0x8000001E) {
           AsmCpuid (0x8000001E, NULL, &CpuidEbx, NULL, NULL);
-          Cpu->CoreCount =
-            (UINT16) DivU64x32 (
-              Cpu->ThreadCount,
-              (BitFieldRead32 (CpuidEbx, 8, 15) + 1)
-            );
+          Cpu->CoreCount = (UINT16)DivU64x32 (Cpu->ThreadCount, (BitFieldRead32(CpuidEbx, 8, 15) + 1));
```

Comment:
Is there any real reason to reformat this line? If anything, you missed a space after `(UINT16)` and after `BitFieldRead32`.

Reply:
i did add a space after them but umm!  its strange that it didn't save the change . 
thanks will correct it !

Reply:
reformatted it to look cleaner.
i took the above written intel code as a reference and it didnt had a space after it so didnt add it .
thanks will correct it